Examples/SourceFile.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftSyntaxBuilder
```

Comment:
A better name for this file would probably be CodeGenerationUsingSwiftSyntaxBuilder.swift.